Library/Formula/ziproxy.rb

```diff
@@ -0,0 +1,29 @@
+class Ziproxy < Formula
+  homepage "http://ziproxy.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/ziproxy/ziproxy/ziproxy-3.3.1/ziproxy-3.3.1.tar.gz"
+  sha1 "7c8db842754d1e3e90b42c3c37f9b15e932a6545"
+
+  def install
+    ENV["LDFLAGS"] = "-lresolv"
```

Comment:
This should be supplied by the buildsystem. Can you please create and submit a patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!